### PR TITLE
Update pronunciation.md

### PR DESCRIPTION
**Pull request by Deczski**
Fixed some grammar

The word "component" was translated to polish "komponent" hence the table was not displayed at all

Added the polish pronounciation and examples
Addes some etymological pronounciation, not all of it because those sounds are not present in polish and I don't have any good examples from other languages

### CODE REVIEW

File: i18n/pl/docusaurus-plugin-content-docs/current/learn/pronunciation.md [hunk 1]
```diff
@@ -13,37 +13,37 @@ Sposób wymawiania języków etnicznych zależy od ich rodzimych użytkowników.
...
+- jak `d` w :pl[**d**om]
+- jak `dż` w :pl[**dż**em]
+- jak `e` w :pl[**e**lf]
+- jak `je` w :pl[**je**ść]
+- jak `f` w :pl[**f**ortepian]
+- jak `g` w :pl[**g**ra]
+- jak `ch` lub `h` w :pl[**ch**usta]
+- jak `i` w :pl[W**i**ktoria]
+- jak `y` w :pl[m**y**sz]
+- jak `k` w :pl[krąg]
```
Comment (noomorph) [inline i18n/pl/docusaurus-plugin-content-docs/current/learn/pronunciation.md:30]:
```suggestion
- jak `k` w :pl[**k**rąg]
```

File: i18n/pl/docusaurus-plugin-content-docs/current/learn/pronunciation.md [hunk 2]
```diff
@@ -13,37 +13,37 @@ Sposób wymawiania języków etnicznych zależy od ich rodzimych użytkowników.
...
+- jako `l` w :pl[luksus]
```
Comment (noomorph) [inline i18n/pl/docusaurus-plugin-content-docs/current/learn/pronunciation.md:31]:
```suggestion
- jako `l` w :pl[**l**uksus]
```

File: i18n/pl/docusaurus-plugin-content-docs/current/learn/pronunciation.md [hunk 3]
```diff
@@ -13,37 +13,37 @@ Sposób wymawiania języków etnicznych zależy od ich rodzimych użytkowników.
...
+- jak `li` w :pl[mi**li**oner]
+- jak `m` w : pl[**m**aks]
+- jak `n` w : pl[**n**]
 - jak `ny` w :en[ca**ny**on]
```
Comment (noomorph) [inline i18n/pl/docusaurus-plugin-content-docs/current/learn/pronunciation.md:35]: Trěba někaky odpovědnik, @Deczski ... 🤔  ne iz anglijskogo jezyka.

Comment (noomorph) [reply i18n/pl/docusaurus-plugin-content-docs/current/learn/pronunciation.md:35]: Ale, čelo, mnogo dobra praca!! 👍

Comment (Deczski) [reply i18n/pl/docusaurus-plugin-content-docs/current/learn/pronunciation.md:35]: - jak `ni` w :pl[ma**ni*ak]

Comment (noomorph) [reply i18n/pl/docusaurus-plugin-content-docs/current/learn/pronunciation.md:35]:
```suggestion
- jak `ny` w :en[kolo**ni**a]
```

**Review by noomorph — APPROVED** [review]

**Pull request merged**